PRESUBMIT.py

```diff
@@ -95,15 +95,12 @@ def CheckWebDevStyle(input_api, output_api):
                                    'tools')):
         # pylint: disable=import-error,import-outside-toplevel
         from web_dev_style import presubmit_support, js_checker
-        with override_utils.override_scope_variable(output_api,
-                                                    'PresubmitPromptWarning',
-                                                    output_api.PresubmitError):
```

Comment:
Chromium checks are too strict and there's no way to ignore them on one by one basis.

To solve this we can either ignore all file or have these checks enabled as warnings so we can see at least some issues and fix those that we can.